nix/packages/default.nix

```diff
@@ -1,15 +1,57 @@
-{
-  perSystem = { pkgs, inputs', ... }: {
-    packages.cardanow = pkgs.writeShellApplication {
-      name = "cardanow";
-      runtimeInputs = [
-        inputs'.cardano-node.packages.cardano-cli
-        inputs'.mithril.packages.mithril-client-cli
-      ];
-      text = ''
-        cardano-cli --version
-        mithril-client --version
-      '';
+{ inputs, ... }: {
+  perSystem = { pkgs, inputs', system, config, ... }:
+    {
+      packages = {
+        cleanup-local-data = pkgs.writeShellApplication {
+          name = "cleanup-local-data";
+          runtimeInputs = with pkgs; [ bash ];
+          text = ''${../../bin/cleanup-local-data.sh} "$@"'';
+        };
+        cleanup-s3-data = pkgs.writeShellApplication {
+          name = "cleanup-local-data";
+          runtimeInputs = with pkgs; [ awscli2 bash jq ];
+          text = ''${../../bin/cleanup-s3-data.sh} "$@"'';
+        };
+
+        upload-data = pkgs.writeShellApplication {
+          name = "upload-data";
+          runtimeInputs = with pkgs; [ bash awscli2 ];
+          text = ''${../../bin/upload-data.sh} "$@"'';
+        };
```

Comment:
Probably I told you to do so but I'm not sure anymore, is the `$@` actually needed?

Reply:
I think it is, we have to propagate args and is seems that's the solution (or at lease one of the solutions)